test/selected-attribute.html

```diff
@@ -66,6 +76,24 @@
 
     });
 
+    suite('changing attrForSelected', function() {
+
+      var s;
+
+      setup(function () {
+        s = fixture('test-attr-change');
+      });
+
+      test('changing selectedAttribute', function() {
+        s.attrForSelected = 'data-y';
+        s.async(function() {
```

Comment:
Just left a comment about using `apply` in IronMultiSelectable behavior. I think that might be the reason. (Or was this here before that?)